lib/tasks/tag_tasks.rake

```diff
@@ -49,6 +49,23 @@ namespace :Tag do
     end
   end
 
+  desc "Reset non canonical warnings"
+  task(reset_non_canonical_warnings: :environment) do
+    choose_not_to_warn = Warning.find_by(name: "Choose Not To Use Archive Warnings")
+    Warning.where(canonical: [false, nil]).each do |warning|
+      if warning.taggings_count.zero?
+        warning.destroy
+      else
+        warning.works.each do |work|
+          work.warnings = choose_not_to_warn if work.warnings.count <= 1
```

Comment:
`if work.warnings.count <= 1` if there's already one warning, do we need to add CNTW?

Reply:
At this point, the non-canonical warning hasn't been converted to a freeform yet, so if the work has one warning, it has to be the non-canonical warning (meaning it doesn't have any of the canonical warning tags).

Reply:
Gotcha, thanks!